package.json

```diff
@@ -10,12 +10,14 @@
     "prebuild": "yarn",
     "build": "ncc build src --out action --minify",
     "lint": "prettier --check \"src/**/*.js\" && eslint src",
+    "format": "prettier --write \"src/**/*.js\"",
```

Comment:
All modern editors have a `run on save` option for prettier nowadays. For the people that forget to enable it, there's a pre-commit hook that'll run for all changed files. 

Knowing that do you think we still need this?

Reply:
the pre-commit hook runs the check but doesn't fix the issues. I have `format on save` configured but somehow one of the files didn't get formatted properly and I was too lazy to find which one so added this. I can remove if you don't think this is gonna be useful for others or going forward though.